tests/jvm/src/test/scala/scala/meta/tests/cli/CliSuite.scala

```diff
@@ -64,38 +64,35 @@ class CliSuite extends FunSuite with DiffAssertions {
       |Uri => HelloWorld.scala
       |Text => non-empty
       |Language => $language
-      |Symbols => 9 entries
+      |Symbols => 7 entries
       |Occurrences => 7 entries
       |Diagnostics => 0 entries
       |Synthetics => 0 entries
       |
       |Symbols:
       |_empty_.HelloWorld. => final object HelloWorld
       |_empty_.HelloWorld.main([Ljava/lang/String;)V. => def main: (args: Array[String]): Unit
-      |  [0:7..0:12): Array => _root_.scala.Array#
-      |  [0:13..0:19): String => _root_.scala.Predef.String#
-      |  [0:23..0:27): Unit => _root_.scala.Unit#
-      |_root_.scala.Array# => final class Array
-      |_root_.scala.Array#`<init>`(I)V. => primaryctor <init>: (_length: Int): Array[T]
-      |  [0:10..0:13): Int => _root_.scala.Int#
-      |  [0:16..0:21): Array => _root_.scala.Array#
-      |  [0:22..0:23): T => _root_.scala.Array#[T]
+      |  args => _empty_.HelloWorld.main([Ljava/lang/String;)V.(args)
+      |  Unit => _root_.scala.Unit#
+      |_empty_.HelloWorld.main([Ljava/lang/String;)V.(args) => param args: Array[String]
+      |  Array => _root_.scala.Array#
+      |  String => _root_.scala.Predef.String#
+      |_root_.scala.Array# => final class Array.{+6 decls}
```

Comment:
What is the use of +6 decls here?

Reply:
To indicate that this information is available if necessary.